doc/strftime_formatting.rdoc

```diff
@@ -461,3 +461,86 @@ The integer width specifier gives a minimum width for the returned string:
   Time.new(2002, 12).strftime('%B')   # => "December" # No width specifier.
   Time.new(2002, 12).strftime('%10B') # => "  December"
   Time.new(2002, 12).strftime('%3B')  # => "December" # Ignored if too small.
+
+== Specialized Format Strings
+
+Here are a few specialized format strings,
+each based on an external standard.
+
+=== HTTP Format
+
+The HTTP date format is based on
+{RFC 2616}[https://datatracker.ietf.org/doc/html/rfc2616],
+and treats dates in the format <tt>'%a, %d %b %Y %T GMT'</tt>;
+see {Formats for Dates and Times}[https://docs.ruby-lang.org/en/master/strftime_formatting_rdoc.html]:
```

Comment:
These "see Formats for Dates and Times" links are just linking back to this document, we can remove these.

Reply:
Removed.